__tests__/seed.test.js

```diff
@@ -22,23 +22,45 @@ describe("/api/topics", () => {
         });
       });
   });
-test('GET 404: user enters invalid endpoint', () => {
+  test('GET 404: user enters invalid endpoint', () => {
     return request(app)
-    .get("/api/invalidURL")
-    .expect(404)
-    .then(({ body }) => {
+      .get("/api/invalidURL")
+      .expect(404)
+      .then(({ body }) => {
         const { message } = body
         expect(message).toBe('Not found')
-    })
+      })
   })
 })
 describe('/api', () => {
   test('GET 200: Returns all available endpoints', () => {
     return request(app)
-    .get('/api')
-    .expect(200)
-    .then(({ body }) => {
-      expect(body).toEqual(endpoints)
-    })
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints)
+      })
+  })
+})
+describe('/api/articles/:article_id', () => {
+  test('GET 200: responds with article that corresponds the article_id', () => {
+    return request(app)
+      .get('/api/articles/4')
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body
+        expect(article).toMatchObject(expect.objectContaining(
+          {
+            title: "Student SUES Mitch!",
+            topic: "mitch",
+            author: "rogersop",
+            body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+            created_at: expect.any(String),
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          }
+        )
+        )
+      })
```

Comment:
great test so far - I'd also like to see some tests for what happens if we try to access a valid but non-existent article ID as well as an invalid article id